app/steps/medicaid/health_disability_member.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Medicaid
+  class HealthDisabilityMember < ManyMembersStep
+    step_attributes(:members)
+
+    def valid?
+      if members.select(&:disabled).any?
+        true
+      else
+        errors.add(:disabled, "Make sure you select at least one person")
```

Comment:
we have this error string a lot, wonder if we should throw it in i18n for share-ablity at some point

Reply:
Yeah I think it's a good idea. Worthwhile to go in the backlog:

https://trello.com/c/SvkqRfzJ/500-pull-make-sure-you-select-at-least-one-person-out-from-the-controllers-and-into-locales-enyml